images/macos/toolsets/toolset-15.json

```
@@ -3,6 +3,17 @@
         "default": "16",
         "x64": {
             "versions": [
+                {
+                    "link": "16.4_beta", 
+                    "version": "16.4-Beta+16F1t",
```

```suggestion
                    "link": "16.4_Release_Candidate", 
                    "version": "16.4_Release_Candidate+16F6",
```

Given RC1 for Xcode 16.4 is already out ?